application/src/main/java/org/togetherjava/tjbot/features/jshell/backend/dto/JShellResult.java

```diff
@@ -1,11 +1,39 @@
 package org.togetherjava.tjbot.features.jshell.backend.dto;
 
+import javax.annotation.Nullable;
+
 import java.util.List;
 
+/**
+ * Result of a JShell eval.
+ * 
+ * @param status status of the snippet
+ * @param type type of the snippet
+ * @param id id of the snippet
+ * @param source source code of the snippet
+ * @param result result of the snippet, nullable
+ * @param exception thrown exception, nullable
+ * @param stdoutOverflow if stdout has overflowed and was truncated
+ * @param stdout what was printed by the snippet
+ * @param errors the compilations errors of the snippet
```

Comment:
"useless javadoc". leaves any reader still puzzled what what means. what is a snippet status? what is a snippet type? what happens if result is null? what if exception is null? who throws the exception? what does it indicate? ...

Reply:
>  what is a snippet status? what is a snippet type?
This is explained in the corresponding classes, check them.

For the rest, I changed things, tell me what you think.